code/modules/status_system/statusSystem.dm

```diff
@@ -1030,20 +1030,30 @@ var/list/statusGroupLimits = list("Food"=4)
 		duration = INFINITE_STATUS
 		maxDuration = null
 		var/mob/living/carbon/human/H
+		var/sleepcount = 5
```

Comment:
Unrelated feature?

Reply:
sleepcount is related - i removed the sleep-in verb from beds and instead added a timer where you lie in a bed for a few seconds you will automatically start sleeping

although now that you point this out that could be a bad idea. i'll think of some other way to do it later